components/AppBanner.vue

```diff
@@ -0,0 +1,41 @@
+<template>
```

Comment:
I don't see this component on the page

Reply:
This markup will be required later
Ref: https://github.com/arcana-network/website/pull/98#discussion_r889811740